config/routes.rb

```diff
@@ -88,7 +88,7 @@ def external_redirect(path_prefix, target)
     # End of public facing routes still rendered by Whitehall
 
     # Routes no longer rendered by Whitehall, but retained to maintain the route helpers
```

Comment:
I see that this comment suggests that none of these paths are rendered by Whitehall, but is there any way we can verify that it is definitely the case, perhaps by looking at what's in the GOV.UK router?

I also see that this comment was introduced in f69c61bc379701483e3d6ae05874f123bed6007e which says:

> These routes exist solely so Rails creates route helpers. They are not served by Whitehall, so we should list them separately from everything else.

So perhaps that's good enough! Also the fact that they were previously responding with a 404 Not Found means we're unlikely to cause a significant problem by making this change.

Reply:
Re-reading my comment I've convinced myself that even if in the unlikely event something was relying on whitehall to render this path removing this route isn't going to make things any worse. So I'll mark this as resolved.